src/esp/physics/PhysicsManager.h

```diff
@@ -420,17 +422,17 @@ class PhysicsManager : public std::enable_shared_from_this<PhysicsManager> {
   // ============ Object Transformation functions =============
 
   /** @brief Set the 4x4 transformation matrix of an object kinematically.
-   * Calling this during simulation of a @ref MotionType::DYNAMIC object is not
-   * recommended.
+   * Calling this during simulation of a @ref esp::physics::MotionType::DYNAMIC
```

Comment:
Strange that you have to fully-qualify these. Does the warning disappear after doing that, at least? If it does, then it's a shitty Doxygen bug (I get these sometimes also).

But generally you shouldn't need to fully-qualify -- if you're referring to something in `esp::physics` from the `esp::physics` namespace, it should be enough to use a relative name, like in C++ itself.

Reply:
> But generally you shouldn't need to fully-qualify -- if you're referring to something in esp::physics from the esp::physics namespace, it should be enough to use a relative name, like in C++ itself.

That's what I thought, but it did seem to address some of the warnings.